source/helpers/debugger.cpp

```diff
@@ -45,8 +45,8 @@ void Debugger::readMemory(void *buffer, size_t bufferSize, u64 address) {
   dmntchtReadCheatProcessMemory(address, buffer, bufferSize);
 }
 
-void Debugger::writeMemory(void *buffer, size_t bufferSize, u64 address) {
-  dmntchtWriteCheatProcessMemory(address, buffer, bufferSize);
+bool Debugger::writeMemory(void *buffer, size_t bufferSize, u64 address) {
+  return R_SUCCEEDED(dmntchtWriteCheatProcessMemory(address, buffer, bufferSize));
```

Comment:
This should rather return result directly and check R_SUCCEEDED later on when the function gets called

Reply:
Agree, I'll change all methods there to get Result, so it can be used later. Okey?